db/eventlog.c

```diff
@@ -152,12 +174,58 @@ void eventlog_perfdata(cson_object *obj, const struct reqlogger *logger) {
     cson_object_set(obj, "perf", perfval);
 }
 
-int write_json( void * state, void const * src, unsigned int n ) {
+int write_json( void * state, const void *src, unsigned int n ) {
     int rc = gzwrite((gzFile) state, src, n);
     bytes_written += rc;
     return rc != n;
 }
 
+int write_logmsg(void *state, const void *src, unsigned int n) {
+    logmsg(LOGMSG_USER, "%.*s", n, src);
+    return 0;
+}
+
+static void eventlog_context(cson_object *obj, const struct reqlogger *logger) {
+    cson_value *contexts = cson_value_new_array();
+    if (logger->ncontext > 0) {
+        cson_array *arr = cson_value_get_array(contexts);
+        cson_array_reserve(arr, logger->ncontext);
+        for (int i = 0; i < logger->ncontext; i++) {
+            cson_value *v = cson_value_new_string(logger->context[i], strlen(logger->context[i]));
```

Comment:
Mike would you mind running clang-format on these files? For example this line is definitely more than 80 characters...